apps/api/src/app/change/usecases/change-enabled/change-enabled.usecase.ts

```diff
@@ -0,0 +1,51 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { ChangeEntityTypeEnum, ChangeRepository } from '@novu/dal';
+import { diffApply } from '../../utils';
+import { ChangeEnabledCommand } from './change-enabled.command';
+import { EnvironmentRepository } from '../../../../../../../libs/dal/src/repositories/environment/environment.repository';
+import { TypeChangeEnabledCommand } from '../type-change-enabled.command';
+import { ChangeEnabledNotificationTemplate } from '../change-enabled-notification-template/change-enabled-notification-template';
+import { ChangeEnabledMessageTemplate } from '../change-enabled-message-template/change-enabled-message-template';
+
+@Injectable()
+export class ChangeEnabled {
```

Comment:
A small questions regarding the naming, maybe worth thinking about more of an action convention, like "EnableChange" rather than something that has happened in the past tense? Or I might be just missing the intention behind this usecase.

Reply:
Or maybe even PromoteChangeToEnvironment